### PR TITLE
Cleanup Logback XML files

### PR DESCRIPTION
**Pull request by awholegunch**
Cleanup XML; makes indents consistent and language similar.

### CODE REVIEW

File: cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml [hunk 1]
```diff
@@ -17,13 +17,11 @@
...
 <!--
-  Logback used by containers that run in YARN eg. Flows, MR, Explore
+  Logback used by containers that run in YARN, such as Explore, Flows, MapReduce, Spark
 -->
 <configuration>
 
-  <!--
-    Disabling some chatty loggers.
-  -->
+  <!-- Disabling some chatty loggers -->
```
Comment (anew) [inline cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml]: Actually, this does not disable them. It "down-tunes" them. Errors still come through.

File: cdap-standalone/src/main/resources/logback.xml [hunk 2]
```diff
@@ -16,18 +16,15 @@
...
   -->
 
-<!--                                                                         -->
-<!-- AppFabric Logging Configuration                                         -->
-<!--                                                                         -->
-<!-- We use the LogBack project for logging in AppFabric. The manual for     -->
-<!-- Logback can be found here: http://logback.qos.ch/manual                 -->
-<!--                                                                         -->
+<!--                                                                                      -->
+<!-- AppFabric Logging Configuration                                                      -->
```
Comment (anew) [inline cdap-standalone/src/main/resources/logback.xml]: I think this should not be AppFabric. It applies to all CDAP system services that run in master or in edge nodes.

File: cdap-standalone/src/main/resources/logback.xml [hunk 3]
```diff
@@ -16,18 +16,15 @@
...
+<!--                                                                                      -->
+<!-- The LogBack project (http://logback.qos.ch/manual) is used for logging in AppFabric. -->
+<!--                                                                                      -->
 
 <configuration>
 
-  <!--
-    Disabling some chatty loggers.
-  -->
+  <!-- Disabling some chatty loggers -->
```
Comment (anew) [inline cdap-standalone/src/main/resources/logback.xml]: see above

**Review by anew — COMMENTED** [review]
only a few comments.

File: cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml (hunk 1 above)
Comment (awholegunch) [reply cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml]: Used "Quieting" instead:
```
  <!-- Quieting some chatty loggers; Errors and Warnings still come through -->

File: cdap-standalone/src/main/resources/logback.xml (hunk 3 above)
Comment (awholegunch) [reply cdap-standalone/src/main/resources/logback.xml]: ibid.

File: cdap-standalone/src/main/resources/logback.xml (hunk 2 above)
Comment (awholegunch) [reply cdap-standalone/src/main/resources/logback.xml]: Replaced that with:
```
<!--                                                                                      -->
<!-- Logging configuration for CDAP System Services on CDAP Master or edge nodes          -->
<!--                                                                                      -->
<!-- The LogBack project (http://logback.qos.ch/manual) is used for CDAP logging          -->
<!--                                                                                      -->

File: cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml [hunk 4]
```diff
@@ -21,7 +21,7 @@
 -->
 <configuration>
 
-  <!-- Disabling some chatty loggers -->
+  <!-- Quieting some chatty loggers; Errors and Warnings still come through -->
```
Comment (anew) [inline cdap-distributions/src/etc/cdap/conf.dist/logback-container.xml]: warnings will not. Only errors.

File: cdap-standalone/src/main/resources/logback.xml [hunk 5]
```diff
@@ -17,14 +17,14 @@
...
+<!-- Logging configuration for CDAP System Services on CDAP Master or edge nodes          -->
 <!--                                                                                      -->
-<!-- The LogBack project (http://logback.qos.ch/manual) is used for logging in AppFabric. -->
+<!-- The LogBack project (http://logback.qos.ch/manual) is used for CDAP logging          -->
 <!--                                                                                      -->
 
 <configuration>
 
-  <!-- Disabling some chatty loggers -->
+  <!-- Quieting some chatty loggers; Errors and Warnings still come through -->
```
Comment (anew) [inline cdap-standalone/src/main/resources/logback.xml]: ditto

**Review by anew — COMMENTED** [review]
two more comments, then LGTM

**Comment by anew** [comment]
LGTM!

**Pull request merged**